### ISSUE TITLE
unable to run it on ubuntu 14.04

### ISSUE THREAD

**Issue by iamkallolpratim** [angular/quickstart#45]
error occurred on npm installing. Following warning message displayed
`npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.0.9
`

**Comment by Foxandxss** [comment]
That library is OSX only. Who's pulling that?

**Comment by iamkallolpratim** [comment]
can u  please suggest me an ubuntu version?

**Comment by Foxandxss** [comment]
The problem is that we are not using that library directly. One of our dependencies are.

**Comment by iamkallolpratim** [comment]
So , how can I setup a angular 2 project in ubuntu 14.04. Please guide me

**Comment by Foxandxss** [comment]
This seems to be an old issue with npm and doesn't seem to be resolved.

Leaving the warning aside, it should work.

Which node & npm do you have?

**Comment by iamkallolpratim** [comment]
Node Version 5.9.0 and NPM version 3.8.3

**Comment by wbeyda** [comment]
Had the same error I restarted the server and browser and it all of a sudden started working.

**Comment by iamkallolpratim** [comment]
@wbeyda , explain me the process please

**Comment by wbeyda** [comment]
@iamkallolpratim , closed the browser tab. Stopped nodejs ctrl+c, then restarted nodejs  "npm start", and opened a new tab 127.0.0.1:3000. Browser opened with the " my first angular app" h1 or whatever it says. I went into main.ts and changed the message to "my second angular app" saved the file and went back to the browser. The browser was updated with "my second angular app" so I knew everything was working. Also make sure your nodejs package is sym linked correctly for Ubuntu. 

sudo ln -s /usr/bin/nodejs /usr/bin/node 

As far as I've read nodejs will always be broke on Ubuntu by default. I'd love to see it fixed by the next LTS release. But for now. It's what we have to do.
